docs/custom-audio.md

```diff
@@ -0,0 +1,15 @@
+# Custom Audio
+To replace a sequences you should only create a zip file with textures to replace at the same path as mk64.o2r. Only wav, mp3, ogg and flac are supported.
```

Comment:
```suggestion
To replace a sequences you should only create a zip file with sequences to replace at the same path as mk64.o2r. Only wav, mp3, ogg and flac are supported.
```
keep the same formulation then https://github.com/HarbourMasters/SpaghettiKart/blob/main/docs/textures-pack.md